src/app/home/home-router.module.ts

```diff
@@ -76,8 +77,14 @@ const alwaysGuardedRoutes = [
 ];
 
 const routes: Routes = [
-  { path: '', component: CommunityComponent, canActivate: [ BetaThenAuthService ] },
-  { path: '',
+  {
+    path: '',
+    component: CommunityComponent,
+    children: [ { path: 'voices/:id', component: NewsListComponent } ],

```

Comment:
This should be added to `backgroundRoute` so the black to blue background doesn't go away when a voices reply is shown.